### PR TITLE
UI improvements 

### PR DESCRIPTION
**Pull request by mikolaj-jalocha**
-  added basic html tags support
- added few more pixel gap between sci circle name and logo 
- every scrollable item  at home screen now has right padding
- fixed shadows on parking's chart

### CODE REVIEW

**Review by simon-the-shark — COMMENTED** [review]
Basically looks good to me, you can read what I'd commented but you can merge if you want.

Only note that Kamil's machine had some weird issues with dependecies and I've made some upgrades and it's possible you'll need to upgrade flutter to newest stable (3.22.smth) after a merge

**Comment by simon-the-shark** [comment]
Let me add a couple of commits to your pull request because I've uploaded those sci circles data and a lot of small ui annoyances were bothering me or few details weren't working as they should. I haven't touched your changes, the above review is still valid and you can do whatever you want there, but if you have a minute, you can take a look at my changes too

File: lib/features/iparking_chart/chart_widget.dart [hunk 1]
```diff
@@ -43,10 +43,8 @@ class ParkingChart extends ConsumerWidget {
           final chartPoints = value.toChartPoints().toList();
           if (chartPoints.isEmpty) {
             return Center(
-              child: Text(
-                context.localize.noChartData,
-                style: context.iParkingTheme.subtitleLight,
-              ),
+              child: Text(context.localize.noChartData,
+                  style: context.iParkingTheme.subtitleLightWithoutShadows),
```
[the remark this answers is not in the document]
Comment (mikolaj-jalocha) [reply lib/features/iparking_chart/chart_widget.dart:47]: Sure, readability matter here much more than slightly better performance. Changed as you proposed.

[the remark this answers is not in the document]
Comment (mikolaj-jalocha) [reply lib/features/iparking_chart/chart_widget.dart:47]: And this extensions's very handy as well :)

**Comment by mikolaj-jalocha** [comment]
> Let me add a couple of commits to your pull request because I've uploaded those sci circles data and a lot of small ui annoyances were bothering me or few details weren't working as they should. I haven't touched your changes, the above review is still valid and you can do whatever you want there, but if you have a minute, you can take a look at my changes too

Thanks for your review, all changes that you proposed were applied. I've also seen your commits, everything looks good for me, along with current ui within our app 😎

**Comment by simon-the-shark** [comment]
You can merge :)

**Pull request merged**